rules-tests/TypeDeclaration/Rector/ClassMethod/ReturnTypeFromStrictNewArrayRector/Source/OrderRepositoryReturnDocblock.php

```diff
@@ -0,0 +1,15 @@
+<?php
+declare(strict_types=1);
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\ReturnTypeFromStrictNewArrayRector\Source;
+
+final class OrderRepositoryReturnDocblock
+{
+    /**
+     * @return array
+     */
+    public function fetchAllForBuyer()
```

Comment:
@TomasVotruba here return docblock be skipped :)

Reply:
Thanks 🙏😁